reference/technology.md

```diff
@@ -58,60 +56,62 @@ Start with a canonical microservice application if you can and confirm that it r
 
 * Level 5: You’re now in full production with GitOps operators and controls, and your release and operations workflows reside within Git.
 
 ## Testing and Issue Detection
 
-* Level 1: When just starting out, much of your testing will be conducted manually on your business application that you’ve identified as your initial production candidate.  With Kubernetes you’ll be focussing on your network connectivity and your ingress, and load balancer configuration, and ensuring you’re able to deploy your applications.  You will have smoke tests, and UAT testing.
+* Level 1: When just starting out, much of your testing will be conducted manually on your business application that you’ve identified as your initial production candidate. With Kubernetes you’ll be focussing on your general network connectivity, and ensuring you’re able to deploy your applications. You will have smoke tests, and UAT testing.
 
-* Level 2: Now that you are into production, you’ll be experimenting with tooling to help with security, policy management, workload misconfigurations, resource requests and limits.
+* Level 2: Now that you are in production, you’ll be experimenting with tooling to help with security, policy management, workload misconfigurations, resource requests, limits and observability, in your staging or development environment.
 
-* Level 3: On the basis of your experiments in the previous level, you’ll be implementing this in production, and including good alerting and good dashboards.  
+* Level 3: On the basis of your experiments in the previous level, you’ll be implementing this in production, and including good alerting and good dashboards, building out your observability capabilities.
 
-* Level 4: Having built up a corpus of issues, you’ll want to work your way through them, ideally fixing or tuning. Some issue remediation may require adjusting your policy-as-code or components of your infrastructure-as-code, as well as your application. Issues may relate to more than one application so you will aggregate across applications to determine trends. These may relate to bugs such as memory leaks, as well as security or policy issues. Your remediation may be to fix them at source, ideally as ‘far left’ as possible, or otherwise building automation capable of fixing them when they occur, and tuning it over time.
+* Level 4: As your environment becomes more complex in production, some issue remediation may require adjusting your policy-as-code or components of your infrastructure-as-code, as well as your application. Issues may relate to more than one application so you will aggregate across applications to determine trends. These may relate to bugs such as memory leaks, as well as security or policy issues. Your remediation may be to fix them at source, ideally as ‘far left’ as possible, or otherwise building automation capable of fixing them when they occur, and tuning it over time.
```

Comment:
there's an "infrastructure-as-code" leftover in the first period. s/infrastructure-as-code/Infrastructure as Code/

Reply:
fixed!